examples/wgpu-examples.ipynb

```diff
@@ -109,7 +109,7 @@
    "name": "python",
    "nbconvert_exporter": "python",
    "pygments_lexer": "ipython3",
-   "version": "3.9.9"
+   "version": "3.9.4"
```

Comment:
This is generated content, a false positive?
```suggestion
   "version": "3.9.9"
```

Reply:
I tried three times to only change the markdown cell... But I guess saving the notebook adds some information about my env. 
Also didn't show up in my diff due to the rich rendering of notebooks. 

will revert